essentials/command_dependencies.dsc

```diff
@@ -0,0 +1,114 @@
+# @ ██████████████████████████████████████████████████████████████████████████████
+# @ ██    command Dependencies | Easy injections to keep error messages consistent
+# % ██
+# % ██  @ command syntax error & stop
+# % ██  | injects an error message telling the player they used
+# % ██  | the command wrong, shows them the command's syntax,
+# % ██  | offers to suggest the command on click and shows
+# % ██  | what they actually typed that was wrong, in-case
+# % ██  | it could have been a typo or to let them see
+# % ██  | for themselves what they typed wrongly
+# - ██  [ Usage ] - inject command_syntax
+command_syntax:
+    type: task
+    debug: false
+    script:
+        - define command "<queue.script.data_key[aliases].first.if_null[<queue.script.data_key[name]>]> "
```

Comment:
should probably keep indentation consistent (throughout files at least)

Reply:
resolved in a queued commit